ros2bag/ros2bag/verb/play.py

```diff
@@ -162,12 +168,20 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '--publish-service-requests', action='store_true', default=False,
             help='Publish recorded service requests instead of recorded service events')
+        parser.add_argument(
+            '--send-actions-as-client', action='store_true', default=False,
+            help='Send the send_goal request, cancel_goal request, and get_result request '
+                 'respectively based on the recorded send_goal, cancel_goal, and get_result '
+                 'event messages. Note that the messages from action\'s "status topic" and '
+                 '"feedback topic" will not be sent because they are expected to be sent from '
+                 'the action server side.')
```

Comment:
actually there is no test for `--publish-service-requests` as cli, probably we can follow this up as service and action playback test with another PR?

Reply:
Yes